.env.sh

```diff
@@ -0,0 +1,3 @@
+export BIGQUERY_PROJECT=pycontw-225217
+export DATATEAM_DISCORD_WEBHOOK=https://discord.com/api/webhooks/777156183055269908/pi3qUddHWyxE1uBpe6CUolJ0q2JUp_aJKvDEgvzXHjrkZwSBl1vSEN1IDqPsvbC8LaEU
+export GOOGLE_APPLICATION_CREDENTIALS=$(pwd)/service-account.json
```

Comment:
I am not sure if it is expected to commit `.env.sh`.

Reply:
yes it's expected, might not be a good practice to disclose our `bigquery project name` 😂 

but should be fine since they don't have `service-account.json`